addon/components/frost-toggle.js

```diff
@@ -130,8 +163,8 @@ export default Component.extend(PropTypeMixin, FrostEventsProxy, {
       }
     },
     /* eslint-enable complexity */
-    noop () {
-      // we have this method to avoid action undefined error
-    }
+
+    /** A no-op action to avoid an action undefined error */
```

Comment:
I respectfully disagree on single line comments. But if we have a majority ruling that even single line comments should have the header and footer, I'll change it.